components/button/skin.css

```diff
@@ -227,10 +234,20 @@ governing permissions and limitations under the License.
   }
 
   &:focus-ring {
-    background-color: var(--spectrum-button-quiet-over-background-background-color-key-focus);
-    border-color: var(--spectrum-button-quiet-over-background-border-color-key-focus);
-    color: inherit;
-    box-shadow: 0 0 0 var(--spectrum-button-primary-border-size-increase-key-focus) var(--spectrum-button-quiet-over-background-border-color-key-focus);
+    background-color: var(--spectrum-button-quiet-over-background-background-color-hover);
+    border-color: var(--spectrum-button-quiet-over-background-border-color-hover);
+    color: var(--spectrum-button-quiet-over-background-text-color-hover);
+    box-shadow: none;
+    &::after {
+      content: '';
+      width: 100%;
+      height: 100%;
+      border: 2px solid var(--spectrum-button-over-background-border-color-key-focus);
+      padding: 4px;
+      position: absolute;
+      top: -6px;
+      border-radius: 22px;
```

Comment:
Put all dimensional values in `index.css`, and create variables for each one, preferably `calc`'d based on existing vars (even if DNA doesn't exist yet) so we have some semantic meaning.